aws/image/Makefile

```diff
@@ -4,20 +4,37 @@
 include ../../podvm/Makefile.inc
 
 
-.PHONY: image clean
+.PHONY: image clean default-vpc
+
+INSTANCE_TYPE ?= t3.small
 
 image: $(IMAGE_FILE)
 
-$(IMAGE_FILE): $(BINARIES) $(FILES)
+$(IMAGE_FILE): setopts $(BINARIES) $(FILES)
 	mkdir -p toupload
-	packer build -var vpc_id=${VPC_ID} \
-		-var subnet_id=${SUBNET_ID} \
-		-var account_id=${AWS_ACCOUNT_ID} \
+	packer build ${OPTS} \
 		-var region=${AWS_REGION} \
 		-var instance_type=${INSTANCE_TYPE} \
-		-var ami_name=${IMAGE_NAME} .
+		-var ami_name=${IMAGE_NAME} ./${PODVM_DISTRO}/
```

Comment:
How PODVM_DISTRO is being used ?

Reply:
just to point to the disto file

Reply:
I see. Thanks

Reply:
Do you plan to cover CentOS in a separate PR ?

Reply:
I haven't planned to add CentOS as part of this PR, this can be done in a following PR